### PR TITLE
[appveyor] Move AppVeyor to Visual Studio 2017, use cached Simbody build

### PR DESCRIPTION
**Pull request by chrisdembia**
@aymanhab's latest 4.0 release betas use VS2017, so it might make sense to use VS2017 on AppVeyor for consistency.

### CODE REVIEW

**Comment by chrisdembia** [comment]
I might try to rope https://github.com/opensim-org/opensim-core/pull/1929 into this PR.

**Comment by chrisdembia** [comment]
This PR is ready for review. @aymanhab would you be interested in reviewing this? This PR contains the move from 4.0beta_nov2017 to master.

File: .travis.yml [hunk 1]
```diff
@@ -317,8 +317,7 @@ deploy:
   skip_cleanup: true
   script: rsync --archive --compress --verbose ~/to_deploy/$ZIPNAME opensim-bot@frs.sourceforge.net:/home/frs/project/myosin/opensim-core/
   on:
-    # TODO temporarily deply on beta branch so GUI doesn't depend directly on master.
-    branch: 4.0beta_nov2017 
+    branch: master
     # Upload for both linux (once) and OSX. Might need to modify the condition
```
Comment (aymanhab) [inline .travis.yml]: :+1:

**Comment by aymanhab** [comment]
Just found the PR on simbody, so LGTM :+!:

**Comment by chrisdembia** [comment]
Thanks @aymanhab . By rule 5, I can merge this.

**Pull request merged**